specification/portalservices/resource-manager/Microsoft.PortalServices/preview/2023-01-01-preview/extensions.json

```diff
@@ -0,0 +1,108 @@
+{
+  "swagger": "2.0",
```

Comment:
SigningOff as Private Repo PR is already SignedOff: [https://github.com/Azure/azure-rest-api-specs-pr/pull/9625](https://github.com/Azure/azure-rest-api-specs-pr/pull/9625).